Source/diablo.cpp

```diff
@@ -587,17 +609,15 @@ void diablo_reload_process(HINSTANCE hInstance)
 			}
 			hWnd = hPrev;
 		}
-		while (1) {
+		do {

```

Comment:
I agree that most other changes are unlikely, but as for the `do-while` I think it could have been used in vanilla instead of `while(1) { if (..) { break } }`